src/SQLStore/EntityStore/TraversalPropertyLookup.php

```diff
@@ -73,46 +71,38 @@ public function fetchFromTable( PropertyTableDef $propertyTableDef, DataItem $da
 			// from the inner join
 			$options['GROUP BY'] = 'p_id';
 
-			$opt = OptionsBuilder::toString( $options );
+			$subquery = $builder->newSubquery();
```

Comment:
I wonder if this should be:

$subquery = $builder->newSubquery()
    ->select( 'p_id' )->from( $propertyTableDef->getName() );

https://github.com/wikimedia/mediawiki/blob/027c882b3502fa08b0ada8f51f133f3ee2083233/includes/block/HideUserUtils.php#L75

Reply:
Oh https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/feat-querybuilder/tests/phpunit/SQLStore/EntityStore/TraversalPropertyLookupTest.php#L31 may need fixing